applications/nrf_desktop/src/modules/ble_state.c

```diff
@@ -271,6 +275,20 @@ static void bt_ready(int err)
 
 	LOG_INF("Bluetooth initialized");
 
+#ifdef CONFIG_BT_LL_NRFXLIB
+	hci_vs_cmd_llpm_mode_set_t cmd = {
+		.enable = 1
+	};
+
+	u8_t hci_err = hci_vs_cmd_llpm_mode_set(&cmd);
```

Comment:
This API is not thread-safe. You should therefore serialize the command and use ```bt_send()```

Reply:
ok. Should I serialize it with what? If it is not thread safe where is the race? I don't have access to LL code so cannot tell and documentation says nothing.